core/src/main/java/nl/nn/adapterframework/stream/Message.java

```
@@ -395,6 +397,14 @@ public String asString(String defaultCharset) throws IOException {
 		if (request instanceof String) {
 			return (String)request;
 		}
+		if(request instanceof Node) {
+			try {
+				request = XmlUtils.nodeToString((Node)request, true);
+			} catch (TransformerException e) {
+				log.warn("Could not convert Document to String", e);
```

Please rethrow the exception, instead of only logging it.